scripts/run-test.sh

```diff
@@ -3,7 +3,8 @@
 # Due to async/await being used in tests, we can't run node version < 8
 echo "Checking node version..."
 node --version | grep v8
-if [[ $? != 0 ]] ; then
+RETURN_CODE=$?
+if [ "${RETURN_CODE}" -ne "0" ]; then
```

Comment:
Previously we were checking the `$TRAVIS_EVENT_TYPE` before executing this script. Are we still wanting to run these tests **only** on `cron` jobs?

Reply:
Ah - reread the PR title. :)

Do we want to to exit with `0` or a failure status when the Node version isn't supported?